guides/common/modules/proc_upgrading-hosts-to-next-major-rhel-release.adoc

```diff
@@ -1,17 +1,31 @@
-[id="Upgrading_Hosts_From_RHEL7_to_RHEL8_{context}"]
-= Upgrading Hosts From RHEL 7 to RHEL 8
+[id="Upgrading_Hosts_to_Next_Major RHEL_release_{context}"]
+= Upgrading Hosts to Next Major {RHEL} Release
 
-You can use a job template to upgrade your {RHEL} 7 hosts to {RHEL} 8.
+You can use a job template to upgrade your {RHEL} hosts to the next major version.
+Below upgrade paths are possible:
+
+* Red Hat Enterprise Linux 7 to Red Hat Enterprise Linux 8
+* Red Hat Enterprise Linux 8 to Red Hat Enterprise Linux 9
 
 .Prerequisites
```

Comment:
I'd do smth like this for the prerequisites to avoid nesting:

```
.Prerequisites
* Ensure that your {RHEL} hosts meet the requirements for the upgrade.
ifndef::orcharhino[]
** For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/planning-an-upgrade_upgrading-from-rhel-7-to-rhel-8[Planning an upgrade] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
** For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/upgrading_from_rhel_8_to_rhel_9/planning-an-upgrade_upgrading-from-rhel-8-to-rhel-9[Planning an upgrade] in the _Upgrading from RHEL 8 to RHEL 9_ guide.
endif::[]
* Prepare your hosts for the upgrade.
ifndef::orcharhino[]
** For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/upgrading_from_rhel_7_to_rhel_8/index#preparing-a-rhel-7-system-for-the-upgrade_upgrading-from-rhel-7-to-rhel-8[Preparing a RHEL 7 system for the upgrade].
** For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/upgrading_from_rhel_8_to_rhel_9/assembly_preparing-for-the-upgrade_upgrading-from-rhel-8-to-rhel-9#preparing-a-rhel-8-system-for-the-upgrade_upgrading-from-rhel-8-to-rhel-9[Preparing a RHEL 8 system for the upgrade].
endif::[]
* Enable remote execution feature on {Project}.
For more information, see xref:Configuring_and_Setting_Up_Remote_Jobs_{context}[].
* Distribute {Project} SSH keys to the hosts that you want to upgrade.
For more information, see xref:Distributing_SSH_Keys_for_Remote_Execution_{context}[].
```